README.md

```diff
@@ -45,12 +44,13 @@ regions:
       password: admin
       tenant: admin
 
-storage:
-  url: http://elastic:9200/
+elastic:
+  urls: ["http://elastic:9200/"]
```

Comment:
you should change format of this field to dicts with keys "host" and "port" (that is standard, unfortunatelly not documented yet)